file-formats/auth/README.md

```diff
@@ -0,0 +1,5 @@
+# AUTH File Format
+
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.auth.json` | 1 | [`zif_aff_auth_v1.intf.abap`](./type/zif_aff_auth_v1.intf.abap) | [`auth-v1.json`](./auth-v1.json) | [`z_aff_example_auth.auth.json`](./examples/z_aff_example_auth.auth.json)
```

Comment:
Example file is missing. Wither we upload it as well, or we remove the link and create an issue in this repository so that it is uploaded later.